helix-term/src/commands.rs

```diff
@@ -625,6 +626,71 @@ fn trim_selections(cx: &mut Context) {
     };
 }
 
+// align text in selection
+fn align_selections(cx: &mut Context) {
+    let align_style = cx.count();
+    if align_style > 3 {
+        cx.editor.set_error(
+            "align only accept 1,2,3 as count to set left/center/right align".to_string(),
+        );
+    }
```

Comment:
There should be a `return` here else it panics because of `unimplemented!` further down.

Reply:
Good catch, thanks!